libpod/container_attach.go

```diff
@@ -64,8 +64,15 @@ func (c *Container) attachContainerSocket(resize <-chan remotecommand.TerminalSi
 	defer controlFile.Close()
 
 	kubecontainer.HandleResizing(resize, func(size remotecommand.TerminalSize) {
+		controlPath := filepath.Join(c.bundlePath(), "ctl")
+		controlFile, err := os.OpenFile(controlPath, unix.O_WRONLY, 0)
```

Comment:
Don't like the naming here - you're shadowing `controlFile` in the function calling `HandleResizing`. `ctrControlFile` might be better?

Reply:
I'll drop the `controlFile` block in the function as it is not used anymore.